account_payment_term_partner_holiday/models/res_partner.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    holiday_ids = fields.One2many(
+        comodel_name="res.partner.holiday",
+        inverse_name="partner_id",
+        string="Holidays",
```

Comment:
We need to be more explicit:

```suggestion
        string="Payment Terms Holidays",
```